### DISCUSSION TITLE
Social bookmarking on the Fediverse

### DISCUSSION THREAD

**Discussion by aschrijver** [goniszewski/grimoire#86] [Ideas]
Cool project, found via [HN](https://news.ycombinator.com/item?id=38419996). Since you are already having account management, [Social bookmarking](https://en.wikipedia.org/wiki/Social_bookmarking) seems like a small step away. Recently [Postmarks](https://techcrunch.com/2023/09/08/with-postmarks-social-bookmarking-is-back-but-this-time-its-built-on-the-fediverse/?guccounter=2) was launched with Fediverse support, and very well received. You may consider adding ActivityPub support and be interoperable with Postmarks and other apps on the Fediverse (like Mastodon and Lemmy). It that doesn't interest you, feel free to close the issue.. I'm just providing a showerthought idea.. :hugs:

**Comment by goniszewski** [comment]
Thanks, that’s a neat idea! I am using Lemmy from time to time and are aware of the benefits coming with ActivityPub, so I’ll support any changes that can bring us closer to the proper integration :)

**Comment by aschrijver** [comment]
I am representing SocialHub, the ActivityPub developer community, where you can find good info and discuss with peers: https://socialhub.activitypub.rocks and would also point to the list of Fediverse Enhancement Proposals (FEP's) at: https://codeberg.org/fediverse/fep

PS. I submitted a post calling for contrib: https://social.coop/@smallcircles/111481622043531308
